src/components/event_preview.js

```diff
@@ -7,6 +7,31 @@ import DataCard from './data_cards';
 // Mostly we expect to deal with the aux_data object format, so if we want to
 // display event options, it's easiest to generate a fake aux_data object
 function makeFakeAuxDataForEventOptions(event) {
+
+  if (event.event_value == "SulisCam"){
```

Comment:
Some background: Event options are baked into the event record (like, a "VEHICLE" event might have an option that says "On deck" or "Off bottom). Auxiliary data is *associated* with an event record but is stored separately in the database.

The point of `makeFakeAuxDataForEventOptions` is to basically split off the baked-in event options into a fake auxiliary data record, so that the rest of the rendering code can treat the two the same way. It's a little bit of a hack but mostly harmless.

I think in this case adding a special case for the SulisCam data is a little bit overloading the purpose of the function. I would spin it off into a separate function, like we do for `reorganizeNavData()` that synthesizes a new auxiliary data record.

https://github.com/WHOIGit/ndsf-sealog-client/blob/ceb065d54856921cf2fa9d0b896095c587ad8abb/src/components/event_preview.js#L146-L147

Aside from this, I'm not entirely sure what the behavior is if you end up with two `vehicleRealtimeFramegrabberData` aux data for a single event. Maybe it should be looking for an existing record, creating if it doesn't exist, and then appending to the `data_array` field.

Reply:
There are also NO helpful event options associated with SulisCam events other than the filename and filepath. This way we're omitting the unhelpful options card that just displays file info and instead feeding `componentForDataSource` a `vehicleRealtimeFramegrabberData` event.

```
{
        "event_value": "SulisCam",
        "event_options": [
            {
                "event_option_value": "/J2-1229/SulisCam/SulisCam.20191001_123550000.jpg",
                "event_option_name": "filename"
            }
        ],
        "ts": "2019-10-01T12:35:50.000Z",
        "event_free_text": "",
        "event_author": "jason",
        "id": "5d93cafafef616043cc13d03"
    }
```

I can break this out into two functions! But this is a short-term fix. My next steps:

1. Run a script to create aux data for every existing SulisCam entry. From there the EventExport API (called every time you click an event in the event list) will gather up said aux data and display it as it does the other camera images. 
2. Delete this Sulis code/function so that we're not displaying the same Sulis image twice.
3. Edit the import script so that it creates aux data AS WELL as event data for Sulises so that we don't see this issue in the future
4. Cleanup the UI from there cuz when you have aux data loading normally you're stuck with an unhelpful "event options" card that displays an overflowing photo filepath. (what we currently see in prod, without the photos)